test/Spec.hs

```diff
@@ -27,6 +28,16 @@ main = hspec $ do
     it "roundtrips correctly" $
       with (Node (TSNode 1 2 3 4 nullPtr nullPtr) nullPtr 1 (TSPoint 2 3) (TSPoint 4 5) 6 7 8) peek `shouldReturn` Node (TSNode 1 2 3 4 nullPtr nullPtr) nullPtr 1 (TSPoint 2 3) (TSPoint 4 5) 6 7 8
 
+  describe "Parser" $ do
+    it "stores a timeout value" $ do
+      parser <- ts_parser_new
+      timeout <- ts_parser_timeout_micros parser
+      timeout `shouldBe` 0
+      ts_parser_set_timeout_micros parser 1000
+      timeout <- ts_parser_timeout_micros parser
+      timeout `shouldBe` 1000
+      ts_parser_delete parser
```

Comment:
This test doesn't do anything interesting *with* the timeout (like parsing and letting it expire) but it seemed worth checking that the numeric argument gets wired through correctly.